### PR TITLE
Add instructions about updating name servers

### PR DESCRIPTION
**Pull request by bbenligiray**
See title. Added both of you because it's probably relevant.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,7 +148,7 @@ services:
 and run `docker-compose run --rm app`, which will create a `./build` directory.
 
 Then, (after installing `ipfs`) run `sudo ipfs add --only-hash --recursive ./build` to get the hash of the build (`sudo` because `build` will likely be owned by root).
-This should be the same as the IPFS hash as the one on the Fleek dashboard and what our ENS/IPNS is pointing towards.
+This should be the same as the IPFS hash as the one on the Fleek dashboard and what our ENS is pointing towards.
```
Comment (bbenligiray) [inline README.md:151]: No longer using IPNS

**Review by andreogle — APPROVED** [review]
👍🏻

File: README.md [hunk 2]
```diff
@@ -107,8 +107,16 @@ branch:
 2. `git merge main`
 3. `git push`
 
-> Note: As of now it's possible to push directly to production, but this will change after
-> https://github.com/api3dao/api3-dao-dashboard/issues/5 is resolved.
+#### Updating the name servers
+
+The primary way to access the DAO dashboard is through the `api3.eth` ENS name, which points directly to the IPFS hash.
+Then, the user can either connect to mainnet on their Metamask and visit `api3.eth/` (the recommended way), or they can visit `https://api3.eth.link/` (this is reported to be down frequently, see [this](https://blog.cloudflare.com/cloudflare-distributed-web-resolver/) for more information).
+We have also forwarded `https://dao.api3.org` to the IPFS hash (using the `dweb.link` gateway), but we do not recommend using this unless necessary.
```
Comment (Siegrift) [inline README.md:114]: You probably want to use the http://api3dao.org/ we talked about in slack.

**Review by Siegrift — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (bbenligiray) [reply README.md:114]: I'll need to ask about it to others first

**Pull request merged**